### PR TITLE
added modal window

### PR DESCRIPTION
**Pull request by BogdanRadetskiy**

### CODE REVIEW

File: crm-client/src/pages/account/UserProject.jsx [hunk 1]
```diff
@@ -1,40 +1,25 @@
...
+            <main>
+                <button className='open-btn' onClick={() => setModalActive(true)}>Project 1</button>
+            </main>
+            <main>
+                <button className='open-btn' onClick={() => setModalActive(true)}>Project 2</button>
+            </main>
+            <main>
+                <button className='open-btn' onClick={() => setModalActive(true)}>Project 3</button>
+            </main>
+            <Modal active={modalActive} setActive={setModalActive}>
```
Comment (wrximpreza) [inline crm-client/src/pages/account/UserProject.jsx]: <Modal active={modalActive} setActive={setModalActive} />

File: crm-client/src/pages/account/UserProject.jsx [hunk 2]
```diff
@@ -1,40 +1,25 @@
-import { Link } from 'react-router-dom'
+import React, {useState} from "react";
+import '../modal/UserProject.css'
+import Modal from "../modal/Modal";
 
+const UserProject = () => {
+    const [modalActive, setModalActive] = useState(true)
```
Comment (wrximpreza) [inline crm-client/src/pages/account/UserProject.jsx]: https://usehooks-ts.com/react-hook/use-boolean

File: crm-client/src/pages/account/UserProject.jsx [hunk 3]
```diff
@@ -1,40 +1,25 @@
...
+    return (
+        <div className='UserProject'>
```
Comment (wrximpreza) [inline crm-client/src/pages/account/UserProject.jsx]: id='UserProject' or className='user-project'

File: crm-client/src/pages/modal/Modal.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./modal.css"
+
+const Modal = ({active, setActive, children}) => {
+    return (
+        <div className={active ? "modal active" : "modal"} onClick={() => setActive(false)}>
```
Comment (wrximpreza) [inline crm-client/src/pages/modal/Modal.jsx]: https://www.npmjs.com/package/classnames

File: crm-client/src/pages/account/UserProject.jsx [hunk 5]
```diff
@@ -1,40 +1,25 @@
-import { Link } from 'react-router-dom'
+import React, {useState} from "react";
+import '../modal/UserProject.css'
+import Modal from "../modal/Modal";
 
+const UserProject = () => {
```
Comment (MaxLogvyniukFive) [inline crm-client/src/pages/account/UserProject.jsx]: Please use function declaration

File: crm-client/src/pages/account/UserProject.jsx [hunk 6]
```diff
@@ -1,40 +1,25 @@
...
+    const [modalActive, setModalActive] = useState(true)
+    return (
+        <div className='UserProject'>
+            <main>
+                <button className='open-btn' onClick={() => setModalActive(true)}>Project 1</button>
```
Comment (MaxLogvyniukFive) [inline crm-client/src/pages/account/UserProject.jsx]: Move onClick from return to function body

File: crm-client/src/pages/account/UserProject.jsx [hunk 7]
```diff
@@ -1,40 +1,25 @@
...
+            </main>
+            <main>
+                <button className='open-btn' onClick={() => setModalActive(true)}>Project 2</button>
```
Comment (MaxLogvyniukFive) [inline crm-client/src/pages/account/UserProject.jsx]: Create one function and reuse it

File: crm-client/src/pages/account/UserProject.jsx [hunk 8]
```diff
@@ -1,40 +1,25 @@
...
+            </main>
+            <main>
+                <button className='open-btn' onClick={() => setModalActive(true)}>Project 3</button>
```
Comment (MaxLogvyniukFive) [inline crm-client/src/pages/account/UserProject.jsx]: -

**Review by MaxLogvyniukFive — CHANGES_REQUESTED** [review]

File: crm-client/src/pages/modal/Modal.jsx [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+            <div className={active ? "modal__content active" : "modal__content"}onClick={e => e.stopPropagation()}>
```
Comment (MaxLogvyniukFive) [inline crm-client/src/pages/modal/Modal.jsx]: Need to add white space "} onClick={e => e.stopPropagation()"

File: crm-client/src/pages/projects/ProjectsInsert.jsx [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+    const { register, handleSubmit, formState: { errors } } = useForm();
+
+    const handleCreateProject = useCallback(async (data) => {
+        const { name } = data
+        const payload = { name }
+
+        try {
+            await api.insertProject(payload);
+
+            window.alert(`Project inserted successfully`)
```
Comment (MaxLogvyniukFive) [inline crm-client/src/pages/projects/ProjectsInsert.jsx]: Nice to change all alerts to [react-toastify](https://www.npmjs.com/package/react-toastify)

**Review by MaxLogvyniukFive — CHANGES_REQUESTED** [review]
Need to update crm-server from express to nest.js

**Review by wrximpreza — CHANGES_REQUESTED** [review]

**Review by RomanPidlisnychyi — APPROVED** [review]

**Pull request merged**